aten/src/ATen/native/quantized/cpu/qembeddingbag.cpp

```diff
@@ -28,8 +29,6 @@
 #include <arm_neon.h>
 #endif
 
-int register_embedding_params();
```

Comment:
One again, might want to double check there isn't a reason for this.

Reply:
Same as above.